### DISCUSSION TITLE
accessing single cell value

### DISCUSSION THREAD

**Discussion by spleshakov** [opensource9ja/danfojs#172] [Q&A] [answered]
Hello,

I was trying to find a way to access one particular cell value by specifying the column and the index. `.loc()` can be used, but that returns a dataframe.

Is there a method that returns a cell's primitive value?

Thank you

**Comment by risenW** [comment, marked as answer]
@spleshakov add a `.values` to the end of your function. Although this returns an array of array, it's closer to what you want. 
We should support this internally. I'll raise an issue. 

```javascript
let data = { "Name": ["Apples", "Mango", "Banana", "Pear"],
            "Count": [21, 5, 30, 10],
            "Price": [200, 300, 40, 250] }

let df = new dfd.DataFrame(data)
let sub_df = df.loc({rows: [0], columns: ["Name"]}).values
console.log(sub_df)
// [[Apples]]
```

Kinda hacky:

```javascript
let sub_df = df.loc({rows: [0], columns: ["Name"]}).values[0][0]
console.log(sub_df)
// Apples

```
